pgjdbc/src/main/java/com/yugabyte/ysql/LoadBalanceManager.java

```diff
@@ -0,0 +1,470 @@
+package com.yugabyte.ysql;
+
+import static com.yugabyte.ysql.LoadBalanceProperties.DEFAULT_FAILED_HOST_TTL_SECONDS;
+import static org.postgresql.Driver.hostSpecs;
+
+import org.postgresql.jdbc.PgConnection;
+import org.postgresql.util.GT;
+import org.postgresql.util.HostSpec;
+import org.postgresql.util.PSQLException;
+import org.postgresql.util.PSQLState;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.Logger;
+
+public class LoadBalanceManager {
```

Comment:
This class mainly has static methods and other utility classes like CloudPlacements etc. Let's see if we can have an alternate name for this.

Reply:
+1 

may be `LoadBalanceStateManagerUtils`

Reply:
+1 

may be `LoadBalanceStateManagerUtils`.

Reply:
`LoadBalanceService` or `LoadBalanceStateManager`? Would like to avoid `Utils` in the name since it is more than that.